app/ui/shared/widgets/overview/cards/base-card.jsx

```diff
@@ -71,53 +72,60 @@ const SUMMARY_STYLE = Style.registerStyle({
   paddingTop: '15px',
 });
 
-const BaseCard = props => {
-  let backgroundImage;
-  let badgeImage;
-
-  if (props.page.location.startsWith(Endpoints.TOFINO_PROTOCOL)) {
-    badgeImage = TOFINO_LOGO;
-  } else {
-    backgroundImage = props.backgroundImage || props.page.meta.image_url;
-    badgeImage = props.badgeImage || props.page.meta.icon_url;
+class BaseCard extends Component {
+  constructor(props) {
+    super(props);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
```

Comment:
From https://facebook.github.io/react/docs/pure-render-mixin.html:

> This only shallowly compares the objects. If these contain complex data structures, it may produce false-negatives for deeper differences.

Is this OK for props.page?